lib/benchpark/cmd/experiment.py

```diff
@@ -18,29 +18,27 @@ def experiment_init(args):
     experiment_spec = benchpark.spec.ExperimentSpec(" ".join(args.spec)).concretize()
     experiment = experiment_spec.experiment
 
-    if args.system:
-        if not os.path.exists(args.system):
-            raise BenchparkError(
-                f"System '{args.system}' does not exist. Run 'benchpark system init --dest {args.system} ...'."
-            )
-        system_file = os.path.join(args.system, "system.pkl")
-        with open(system_file, "rb") as f:
-            system_spec = pickle.load(f)
+    if not os.path.exists(args.system):
+        raise BenchparkError(
+            f"System '{args.system}' does not exist. Run 'benchpark system init --dest {args.system} ...'."
+        )
+    system_file = os.path.join(args.system, "system.pkl")
+    with open(system_file, "rb") as f:
+        system_spec = pickle.load(f)
 
-        experiment.system_spec = system_spec
+    experiment.system_spec = system_spec
 
-    if args.basedir:
-        base = args.basedir
-        expdir = str(hash(experiment_spec))
-        destdir = os.path.join(base, expdir)
-    elif args.dest:
-        destdir = args.dest
+    if args.dest:
+        experiment_component = args.dest
     else:
-        raise ValueError("Must specify one of: --dest, --basedir")
+        experiment_component = experiment_spec.name
+
+    destdir = os.path.join(args.system, experiment_component)
 
     try:
         os.makedirs(destdir)
         experiment.write_ramble_dict(f"{destdir}/ramble.yaml")
+        print(f"Run `benchpark setup {destdir} ...` to generate Ramble workspace")
```

Comment:
Since the only option is the `experiments_root`, I think this is clearer than `...` (I think ellipsis is fine for system since there can be varying amount of variants). Or maybe `benchpark setup {destdir} EXPERIMENTS_ROOT`?
```suggestion
        print(f"Run `benchpark setup {destdir} <experiments_root>` to generate Ramble workspace")
```